openff/interchange/components/_packmol.py

```diff
@@ -893,23 +897,50 @@ def solvate_topology(
         [atom.mass for atom in cl.atoms],
     )
     water_mass = sum([atom.mass for atom in water.atoms])
-    molarity_pure_water = Quantity(55.5, "mole / liter")
+    molarity_pure_water = Quantity(55.4, "mole / liter")
+
+    # Compute the number of salt ions to add for bulk salt
+    # If ionic strength is zero, then the solute_ion_ratio needed for SLTCAP is undefined
+    if nacl_conc == Quantity(0, "mole / liter"):
+        nacl_to_add = 0
+        water_mass_to_add = solvent_mass
+
+    else:
+        # Compute the number of salt "molecules" to add from the mass and concentration
+        # for a neutral solute
+        neutral_nacl_mass_fraction = (nacl_conc * nacl_mass) / (molarity_pure_water * water_mass)
+        neutral_nacl_mass_to_add = solvent_mass * neutral_nacl_mass_fraction
+        neutral_nacl_to_add = (neutral_nacl_mass_to_add / nacl_mass).m_as("dimensionless").round()
+
```

Comment:
When this is 0.0 (after rounding) the following lines crash with NaN - should we just

```
        nacl_to_add = 0
        water_mass_to_add = solvent_mass
```
like above?